tests/runtestcase

```diff
@@ -3,7 +3,7 @@ TEST_TIMEOUT=${TEST_TIMEOUT:-5m}
 SETUP_TIMEOUT=${SETUP_TIMEOUT:-2m}
 
 export CLEANUPDBDIR=${CLEANUPDBDIR:-1}
-export PATH="${SRCHOME}/:${PATH}"
+export PATH="${TESTDIR}/:${PATH}"
```

Comment:
I see that we are exporting PATH at a couple of places. How about doing it in testcase.mk (top-level) after we have generated a random test directory name?

Reply:
putting it in testcase.mk gave error about recursive referencing PATH, so make does not like it there.